tests/vmi_configuration_test.go

```diff
@@ -1692,24 +1692,21 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				tests.UpdateKubeVirtConfigValueAndWait(*config)
 
 				By("Creating a new VMI")
-				var vmi = tests.NewRandomVMI()
+				vmi := tests.NewRandomVMI()
 				// Runtime class related warnings are expected since we created a fake runtime class that isn't supported
 				wp := watcher.WarningsPolicy{FailOnWarnings: true, WarningsIgnoreList: []string{"RuntimeClass"}}
 				vmi = tests.RunVMIAndExpectSchedulingWithWarningPolicy(vmi, 30, wp)
 
 				By("Checking for presence of runtimeClassName")
 				pod := tests.GetPodByVirtualMachineInstance(vmi)
+				Expect(pod.Spec.RuntimeClassName).ToNot(BeNil())
```

Comment:
nit: might be me (and this might be opinionated ...), but I prefer to read `NotTo` instead.

Reply:
Well, most of the usages in this file use `ToNot`, so I prefer to keep with that :)